### ISSUE TITLE
[3.13.0 RC4] Avatars in a shared file

### ISSUE THREAD

**Issue by Valdnet** [nextcloud/android#6682]
### Steps to reproduce
1. Select "All files" from the menu.
2. Share the file with another user.
3. Return to the "All files" menu.

### Expected behaviour
- Avatars of shared users should contain the correct first letter of the username.

**In my opinion, if the user has his avatar set, it should be displayed for shared files.**

### Actual behaviour
- The avatars of shared users contain an invalid letter of the name.

![obraz](https://user-images.githubusercontent.com/47037905/89738528-a4436680-da79-11ea-807c-a028bb28b3f8.png)

After selecting file sharing details, the letter in the avatar is correct.

![obraz](https://user-images.githubusercontent.com/47037905/89738570-13b95600-da7a-11ea-9e7f-6fe767c7dc16.png)

### Environment data
Android version: 10

Nextcloud app version: 3.13.0 RC4 and Dev 20200809

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky I can reproduce this (to some extend).

What I tested (2 users with usernames and name differing the name being first/last name leading to server-side 2-letter avatars, one user with a uploaded image)

* Sharing details view only shows the single-letter avatar (thus must be client-side generated)
* oc file list, initially shows single-letter with some folder refresh app restarts shows the 2-letter avatars
* _no_ display of the image avatar

**Comment by tobiasKaminsky** [comment]
I am totally out of clue :/
I fear we have to postpone this to 3.13.1, but I will mark it as "high".

**Comment by Valdnet** [comment]
> I fear we have to postpone this to 3.13.1

@tobiasKaminsky Version 3.13.1 released, and there are no correct avatars 😁.

**Comment by tobiasKaminsky** [comment]
@Valdnet can you test it with https://github.com/nextcloud/android/pull/7098 ?

**Comment by Valdnet** [comment]
@tobiasKaminsky In the test version, the letters in the avatars are displayed correctly 👍.
Is it possible to show avatar pictures in sharing details? Same as they appear in the list of shared files.

**Comment by tobiasKaminsky** [comment]
> @tobiasKaminsky In the test version, the letters in the avatars are displayed correctly +1.

nice :+1: 



> Is it possible to show avatar pictures in sharing details? Same as they appear in the list of shared files.

It should already be correctly shown, as both use the same function to generate avatar.
Can you post a sample?


> Off topic:

Please create a new issue.

**Comment by Valdnet** [comment]
@tobiasKaminsky This is what it looks like:

View from the file list:

![obraz](https://user-images.githubusercontent.com/47037905/97429753-25432900-1918-11eb-8e51-75d6149a8969.png)

View with file sharing details:

![obraz](https://user-images.githubusercontent.com/47037905/97430351-114bf700-1919-11eb-9a5e-980b9c19b534.png)

**Comment by tobiasKaminsky** [comment]
Hm. What is GPS 2? A regular user, a federated user?

**Comment by Valdnet** [comment]
This is a regular user 😉.

**Comment by tobiasKaminsky** [comment]
@Valdnet 
There is an APK file in this PR #7271, which you can install in parallel to your existing Nextcloud app.

**Comment by Valdnet** [comment]
@tobiasKaminsky https://github.com/nextcloud/android/pull/7271#issuecomment-724642883

Avatars are displayed correctly 🥇.
